### DISCUSSION TITLE
The status of Jupyter extension is "installing" forever

### DISCUSSION THREAD

**Discussion by Guardian-of-Lizi** [microsoft/vscode-jupyter#10181] [Questions and Answers] [answered]
Dear community, 

I have been using vscode + Jupyter without any problem for a year. Several weeks ago, suddenly I could not run my notebooks as it always said "Install kernels from the market" instead of listing the available kernels. In the meanwhile, the status of Jupyter extension became "installing". However, python worked well in prompt. Just the vscode + Jupyter combination stopped working.

So I completely removed vscode and all the extensions in the .vscode folder and reinstalled vscode. Then when I installed the Jupyter extension, it got stuck at "installing" forever, even though the other extensions were installed successfully (python and Jupyter Keymap + Notebook Renderers).

![image](https://user-images.githubusercontent.com/90312479/171025728-acf0aa88-d7a5-4c2a-9c6a-a95002c3099b.png)

I am behind a proxy, but given the other extensions can be installed without problem, I don't know whether this is the cause.

Here are some information:
- VS code:
    Version: 1.67.2 (user setup)
    Commit: c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5
    Date: 2022-05-17T18:15:52.058Z
    Electron: 17.4.1
    Chromium: 98.0.4758.141
    Node.js: 16.13.0
    V8: 9.8.177.13-electron.0
    OS: Windows_NT x64 10.0.19044
- Jupyter extension:
    v2022.4.1021342353

Any idea is appreciated!

**Comment by rchiodo** [comment, marked as answer]
This sounds like a VS code problem to me. I recommend asking there: https://github.com/microsoft/vscode/issues

**Reply by greazer** [reply]
When you add the issue to the vscode issues list, please add the following information to it. Someone there will be able to help you out. 

* Enable log level to trace using the command F1 > Set Log Level
* Try installing the Jupyter extension
* Share Log (Window) and Log (Shared) logs using the command F1 > Open View... > Log (...)

**Reply by Guardian-of-Lizi** [reply]
Thanks for the tips!
